lib/cocoapods/command/update.rb

```diff
@@ -23,13 +23,15 @@ def self.options
         [
           ['--sources=https://github.com/artsy/Specs,master', 'The sources from which to update dependent pods. ' \
            'Multiple sources must be comma-delimited. The master repo will not be included by default with this option.'],
+          ['--skip=podName', 'Pods to skip during update. Multiple sources must be comma-delimited.'],
```

Comment:
I experimented with:
```
pod update --exclude=Alamofire
pod update --ignore=Alamofire
pod update --skip=Alamofire
```
Skip sounded like the best to communicate the intent. But happy to accept suggestions.

Reply:
this still says "multiple sources" which is not relevant to this command

Reply:
🤦‍♂️